plugin-server/functional_tests/e2e.test.ts

```diff
@@ -541,9 +577,9 @@ const fetchSessionRecordingsEvents = async (clickHouseClient: ClickHouse, teamId
 const fetchPluginLogEntries = async (clickHouseClient: ClickHouse, pluginConfigId: number) => {
     const { data: logEntries } = (await clickHouseClient.querying(`
         SELECT * FROM plugin_log_entries
-        WHERE plugin_config_id = ${pluginConfigId}
+        WHERE plugin_config_id = ${pluginConfigId} AND source = 'CONSOLE'
     `)) as unknown as ClickHouse.ObjectQueryResult<PluginLogEntry>
-    return logEntries
+    return logEntries.map((entry) => ({ ...entry, message: JSON.parse(entry.message) }))
```

Comment:
why should we enforce that logs in these tests be JSON?

Reply:
This was to make it easy to apply expectations but yeah there's no real need for it.